code/modules/mob/living/carbon/human/species.dm

```diff
@@ -154,6 +157,10 @@
 	H.visible_message("<span class='notice'>[H] hugs [target] to make [t_him] feel better!</span>", \
 					"<span class='notice'>You hug [target] to make [t_him] feel better!</span>", null, 4)
 
+/datum/species/proc/random_name(var/gender)
+	if(gender==FEMALE)	return capitalize(pick(first_names_female)) + " " + capitalize(pick(last_names))
```

Comment:
SMASH THAT MFING ENTER BUTTON

Reply:
This is cut and pasted shit CM (Bay?) code. I can fix it.